### DISCUSSION TITLE
motioncapture optitrack problem (rviz2-stereo is not supported)

### DISCUSSION THREAD

**Discussion by eunseo01** [bitcraze/discussions#850] [Q&A] [answered]
﻿Hi, we are a group of people who want to do CrazySwarm2 using motion capture (OptiTrack) and the multi-ranger deck. Currently, when we execute the ROS launch, we encounter the following failure, and the Crazyflie is not visible in RViz. Can you help us figure out what the problem might be?
![스크린샷, 2023-08-02 13-02-16](https://github.com/bitcraze/discussions/assets/107001681/715e6abd-75f6-4099-b99c-3b08065efefa)
![스크린샷, 2023-08-02 13-02-19](https://github.com/bitcraze/discussions/assets/107001681/fcc65796-0de5-40f1-be7b-59fedd0ae8fe)
![스크린샷, 2023-08-02 13-05-14](https://github.com/bitcraze/discussions/assets/107001681/37181441-1822-4adb-82ee-c6fa288a2193)

**Comment by knmcguire** [comment, marked as answer]
I see you double posted this question on the Crazyswarm2 forum which is a better place to answer this question. https://github.com/IMRCLab/crazyswarm2/discussions/269
